### DISCUSSION TITLE
2007/12/29/%E6%9F%A5%E7%9C%8B%E4%B8%80%E6%AE%B5%E4%BB%A3%E7%A0%81%E7%9A%84%E8%BF%90%E8%A1%8C%E6%97%B6%E9%97%B4

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#200] [Announcements]
# 2007/12/29/%E6%9F%A5%E7%9C%8B%E4%B8%80%E6%AE%B5%E4%BB%A3%E7%A0%81%E7%9A%84%E8%BF%90%E8%A1%8C%E6%97%B6%E9%97%B4

有时候调试或者测试一段程序的时候，需要查看这段程序到底运行了多少时间。这里的时间还不同于程序所消耗的CPU时间，而就是指程序运行一遍直观的耗时。一般，编写一段简单程序就可以完成这个工作了，如下图所示。使用一个顺序结构，在被测程序开始前，记录下当前时间，程序运行结束在查看一下当前时间，差值就是程序运行的时间。

https://blog.qizhen.xyz/2007/12/29/%E6%9F%A5%E7%9C%8B%E4%B8%80%E6%AE%B5%E4%BB%A3%E7%A0%81%E7%9A%84%E8%BF%90%E8%A1%8C%E6%97%B6%E9%97%B4

**Comment by ruanqizhen** [comment]
之前博客下的留言：

greentea
2007年12月30日 16:52 编辑
我是LabVIEW初学者，有个slope的LabVIEW实例——examples\\apps\\demos\\slope.vi，我没有看懂，希望能帮我看看，谢谢。

回复
Arthur
2007年12月29日 15:34 编辑
GPan说的是不是第二种方法提高了优先级？

回复
Jing
2007年12月29日 15:30 编辑
第二种方法会影响里面循环的执行速度，不能用来计时。对不？

回复
